src/ucp/api/ucp.h

```diff
@@ -495,21 +495,21 @@ typedef enum {
  * @brief Atomic operation requested for ucp_atomic_op_nbx
  *
  * This enumeration defines which atomic memory operation should be
- * performed by the ucp_atomic_op_nbx routine. Some of these (with _POST_ in the
- * name) are non-fetching atomics and will not result in a request handle.
- * Others (with _FETCH_ in the name) will fetch data from the remote node.
+ * performed by the ucp_atomic_op_nbx routine. Some of these (@a add, @a and,
+ * @a or, @a xor) are non-fetching atomics and will not result in a request
+ * handle. Others will fetch data from the remote node.
  */
 typedef enum {
-    UCP_ATOMIC_OP_FETCH_FADD,  /**< Atomic Fetch and add    */
-    UCP_ATOMIC_OP_FETCH_SWAP,  /**< Atomic swap             */
-    UCP_ATOMIC_OP_FETCH_CSWAP, /**< Atomic conditional swap */
-    UCP_ATOMIC_OP_FETCH_FAND,  /**< Atomic Fetch and and    */
-    UCP_ATOMIC_OP_FETCH_FOR,   /**< Atomic Fetch and or     */
-    UCP_ATOMIC_OP_FETCH_FXOR,  /**< Atomic Fetch and xor    */
-    UCP_ATOMIC_OP_POST_ADD,    /**< Atomic add */
-    UCP_ATOMIC_OP_POST_AND,    /**< Atomic and */
-    UCP_ATOMIC_OP_POST_OR,     /**< Atomic or  */
-    UCP_ATOMIC_OP_POST_XOR,    /**< Atomic xor */
+    UCP_ATOMIC_OP_FADD,  /**< Atomic Fetch and add    */
+    UCP_ATOMIC_OP_SWAP,  /**< Atomic swap             */
+    UCP_ATOMIC_OP_CSWAP, /**< Atomic conditional swap */
+    UCP_ATOMIC_OP_FAND,  /**< Atomic Fetch and and    */
+    UCP_ATOMIC_OP_FOR,   /**< Atomic Fetch and or     */
+    UCP_ATOMIC_OP_FXOR,  /**< Atomic Fetch and xor    */
+    UCP_ATOMIC_OP_ADD,   /**< Atomic add */
```

Comment:
need to unite UCP_ATOMIC_OP_FADD+UCP_ATOMIC_OP_ADD, UCP_ATOMIC_OP_FOR+UCP_ATOMIC_OP_OR etc